builtin/providers/aws/resource_aws_route53_zone.go

```diff
@@ -64,6 +86,7 @@ func resourceAwsRoute53ZoneCreate(d *schema.ResourceData, meta interface{}) erro
 	zone := cleanZoneID(*resp.HostedZone.ID)
 	d.Set("zone_id", zone)
 	d.SetId(zone)
+	d.Set("vpc_region", req.VPC.VPCRegion)
```

Comment:
> I guess this d.Set should be up in the if block above so it only occurs when using the vpc fields. Does that make sense?

It may make more sense to set it in the `Read` method